_includes/head.html

```diff
@@ -59,3 +59,6 @@
     <link rel="alternate" type="text/html" href="{{ site.baseurl }}/assets/css/styles.css">
   {% endif %}
 {% endif %}
+
+<!-- style guide -->
+<script src="{{ site.baseurl }}/assets/js/styleguide.js"></script>
```

Comment:
It looks like you can remove this script reference here and place it back in the `_includes/scripts.html` file.

Reply:
Done!